cypress/e2e/3-auth/invalid.login.cy.js

```diff
@@ -0,0 +1,23 @@
+it("should show an error message upon a failed login attempt", () => {
+  cy.visit("http://127.0.0.1:5500");
```

Comment:
```suggestion
  cy.visit("/");
```

This uses the built in server from cypress